<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QListWidget" name="sidebarWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="resourcecs.qrc">
         <normaloff>:/resources/icons/home.png</normaloff>:/resources/icons/home.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="resourcecs.qrc">
         <normaloff>:/resources/icons/visualizer.png</normaloff>:/resources/icons/visualizer.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="resourcecs.qrc">
         <normaloff>:/resources/icons/settings.png</normaloff>:/resources/icons/settings.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="HomePage" name="homePage"/>
      <widget class="UnityPage" name="visualizerPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="SettingsPage" name="settingsPage"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomePage</class>
   <extends>QWidget</extends>
   <header>homepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UnityPage</class>
   <extends>QWidget</extends>
   <header>unitypage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsPage</class>
   <extends>QWidget</extends>
   <header location="global">settingspage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resourcecs.qrc"/>
 </resources>
 <connections/>
</ui>
